include/app/tabs/camera.hpp

```diff
@@ -1,35 +1,60 @@
-#pragma once
-
-#include <QLabel>
-#include <QMediaPlayer>
-#include <QStackedLayout>
-#include <QString>
-#include <QVideoWidget>
-
-#include "app/config.hpp"
-#include "app/theme.hpp"
-
-class CameraTab : public QWidget {
-    Q_OBJECT
-
-   public:
-    CameraTab(QWidget *parent = nullptr);
-
-   private:
-    QWidget *connect_widget();
-    QWidget *camera_widget();
-    QWidget *input_widget();
-    void connect_stream();
-    void update_status(QMediaPlayer::MediaStatus media_status);
-
-    Theme *theme;
-    Config *config;
-    QLabel *status;
-    QMediaPlayer *player;
-    QString url;
-
-   signals:
-    void connected();
-    void disconnected();
-};
-
+#ifndef CAMERA_HPP_
```

Comment:
make this `#pragma once`

Reply:
I'm happy to do whatever but it looks like all of the other files use this format.

Reply:
yeah we did back in intelligent auto, if you look now you'll see it was changed to pragmas

Reply:
Ah, gotcha. Pragma is better anyway.